packages/fxa-auth-server/lib/routes/subscriptions/stripe-webhook.ts

```diff
@@ -394,6 +393,11 @@ export class StripeWebhookHandler extends StripeHandler {
       if (err.output && typeof err.output.payload === 'object') {
         err.output.payload = { ...err.output.payload, eventId: event.id };
       }
+      // If the customer has been deleted, then there's insufficient information to
+      // send the email. We can't do anything about it, so we'll ignore the error.
+      if (err.errno === error.ERRNO.UNKNOWN_SUBSCRIPTION_CUSTOMER) {
+        return;
+      }
```

Comment:
This is not a blocker but since I do not know what our rule is regarding multiple return statements, is there any benefit to havign this written so that we report the sentry error if it is not this type of error since that is the only difference in the control logic? i.e. 
```
if (err.errno !== error.ERRNO.UNKNOWN_SUBSCRIPTION_CUSTOMER) {
  reportSentryError(err, request);
}
```